recipes/5-service/Recipe.md

```diff
@@ -1,7 +1,70 @@
 # Service Recipe
 
+In this recipe we will convert the classes in the [service](../../src/main/java/nl/rabobank/kotlinmovement/recipes/service) package to
+kotlin.
 
+There is 1 service class, 1 object mapper class and 1 exception class that 
```

Comment:
Deze zin moet nog worden afgemaakt :)

Reply:
Fixed in 787881cec4653d9c956fb90aef2945a7be60974d